package-lock.json

```diff
@@ -5692,9 +5909,10 @@
       }
     },
     "@rocket.chat/apps-engine": {
-      "version": "git+https://github.com/WideChat/Rocket.Chat.Apps-engine.git#7777d4b7702500a3c2ff28aaae9fa8963dba109f",
-      "from": "git+https://github.com/WideChat/Rocket.Chat.Apps-engine.git#master",
+      "version": "git+https://github.com/WideChat/Rocket.Chat.Apps-engine.git#35877292b61dea5345685fe2879906b4f1df2977",
+      "from": "git+https://github.com/WideChat/Rocket.Chat.Apps-engine.git#dialogflow_branch",
```

Comment:
NOTE: this is fine for testing, but we need to be careful that we revert of fix this before we merge.

Reply:
Fixed dependencies.